src/services/orchestrator.ts

```
@@ -0,0 +1,69 @@
+      for (const pr of mergedPRs) {
+        try {
+          const summary = await this.openAIService.generatePRSummary(
+            pr.body || "",
+            pr.title
+          );
+          const tweet = await this.openAIService.generateTweetText(summary);
+          await submitMessage(tweet);
+          console.log(`Successfully processed PR #${pr.number}`);
+        } catch (error) {
+          console.error(`Error processing PR #${pr.number}:`, error);
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)